.github/workflows/check.yml

```diff
@@ -68,6 +68,12 @@ jobs:
           /home/runner/.cargo/bin/rustup show
       - name: Run Test (with coverage)
         run: |
+         # TODO: replace with out CI docker image, first attempt given:
```

Comment:
By the way codecov has been stabilized as of 1.60 if the images are updated to latest stable we don't need nightly anymore here